src/OrchardCore.Modules/OrchardCore.Localization/Assets.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "inputs": [
```

Comment:
We may need to support RTL too

Reply:
Looks good to me:

![image](https://user-images.githubusercontent.com/1165805/58740556-d2351180-83c5-11e9-8a16-34742c8d3835.png)